lib/layouts/PageHeading.js

```diff
@@ -11,14 +11,15 @@ class PageHeading extends Component {
       breadcrumbLinks = null,
```

Comment:
We don't need to sync `PageHeading`. We should be syncing `Header` from layouts.

Reply:
@yedhink Let's handle layouts separately and not on this PR.

cc @goutham-subramanyam